demo/api/src/news/blocks/news-detail.block.ts

```diff
@@ -0,0 +1,37 @@
+import { BlockData, BlockField, BlockIndexData, BlockInput, createBlock, inputToData } from "@comet/blocks-api";
+import { IsOptional, IsUUID } from "class-validator";
+
+class NewsDetailBlockData extends BlockData {
```

Comment:
We already have [NewsLinkBlock](https://github.com/vivid-planet/comet/blob/next/demo/api/src/news/blocks/news-link.block.ts), should we use that for this use case? IMO it's a more realistic use case.

Reply:
That NewsDetailBlock is perfect use case, for a NewsLink you'll never use such a loader

Reply:
Yes

Reply:
If we want to use transformToPlain for use cases like this we need to simplify its usage. Currently we need a lot of complicated stuff in application code to achieve that.